src/schema/term.rs

```diff
@@ -143,6 +150,13 @@ impl Term {
         self.0.extend(bytes);
     }
 
+    /// Sets the value of a `Vector` field.
+    pub fn set_vector(&mut self, vector: &Vec<f32>) {
```

Comment:
I think you can use a slice here

Reply:
Ah yes! Clippy usually tells me to change it! I was just not using it yet.